Specs/Scene/SkyAtmosphereSpec.js

```diff
@@ -55,6 +55,34 @@ defineSuite([
         s.destroy();
     });
 
+    it('draws sky with setDayNight(true) ', function() {
```

Comment:
Just to keep with the style, word this as "draws sky with setDayNight set to true" instead. (And below)